x-pack/plugins/apm/public/components/alerting/fields.tsx

```diff
@@ -21,6 +21,9 @@ import {
 import { SuggestionsSelect } from '../shared/suggestions_select';
 import { PopoverExpression } from './service_alert_trigger/popover_expression';
 
+const start = moment().subtract(24, 'h').toISOString();
```

Comment:
this needs to be inside of the render function, otherwise it's only initialized when the javascript loads in the browser.

Reply:
what I was thinking 🙈 Updating it